packages/botonic-react/tests/components/text-serialization.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Text } from '../../src/components/text'
+import { Button } from '../../src/components/button'
+
+describe('Text serialization', () => {
+  it('Serializes a simple text', () => {
```

Comment:
these 4 functions can be implemented by a single test.each function

Reply:
added `ESCAPED_LINE_BREAK ` and refactored with `test.each` 👍